bundler/lib/bundler/cli/plugin.rb

```diff
@@ -14,7 +14,7 @@ class CLI::Plugin < Thor
     method_option "git", :type => :string, :default => nil, :banner =>
       "URL of the git repo to fetch from"
     method_option "local_git", :type => :string, :default => nil, :banner =>
-      "Path of the local git repo to fetch from"
+      "Path of the local git repo to fetch from (deprecated)"
```

Comment:
Can we also add a `SharedHelpers.major_deprecation` call, similarly to other flag deprecations we have in place in this file?

Reply:
Done (though over in plugin/installer.rb, where we implement the backwards compatibility)